### PR TITLE
Clean fs for every test

### PR DESCRIPTION
**Pull request by phlax**
This PR:

- adds a session-scoped fixture for use of PootleEnv
- copies out the contents of PootleEnvs translation_dir and set the POOTLE setting
- updates tests/fixtures to make use of new fs env

### CODE REVIEW

File: pytest_pootle/fixtures/models/store.py [hunk 1]
```diff
@@ -275,35 +273,6 @@ def _make_member_updates(store, member):
     _mark_unit_fuzzy(store.units[0], member)
 
 
-@pytest.fixture(scope='session')
```
Comment (ta2-1) [inline pytest_pootle/fixtures/models/store.py]: @phlax can we put this change to https://github.com/translate/pootle/pull/4585/commits/3738cca454a24f2163d7e38ae42a1a0ffaf5bb2b commit to show that it was moved?

File: pytest_pootle/fixtures/models/translation_project.py [hunk 2]
```diff
@@ -23,7 +23,8 @@ def _require_tp(language, project):
     """Helper to get/create a new translation project."""
     from pootle_translationproject.models import create_translation_project
 
-    return create_translation_project(language, project)
+    tp = create_translation_project(language, project)
+    return tp
```
Comment (ta2-1) [inline pytest_pootle/fixtures/models/translation_project.py]: is it necessary?

File: pytest_pootle/fixtures/models/store.py (hunk 1 above)
Comment (phlax) [inline pytest_pootle/fixtures/models/store.py]: yep

File: pytest_pootle/fixtures/import_export_fixtures.py [hunk 3]
```diff
@@ -31,40 +31,32 @@ def file_import_failure(request):
 
 
 @pytest.fixture
-def ts_directory(request, tmpdir):
-    """Sets up a tmp directory with test ts files."""
+def ts_directory(po_directory, request, tmpdir, settings):
```
Comment (ta2-1) [inline pytest_pootle/fixtures/import_export_fixtures.py]: @phlax I dropped `po_directory` fixture from here and all tests passed.

Comment (phlax) [inline pytest_pootle/fixtures/import_export_fixtures.py]: cool - we can remove i think i just added to be sure that the env was to true to what would happen - ie the po files would already be there when the ts projects were set up.

Comment (phlax) [inline pytest_pootle/fixtures/import_export_fixtures.py]: ...but i dont see it making much diff either way

Comment (phlax) [inline pytest_pootle/fixtures/import_export_fixtures.py]: cool as agreed - kept - but docstring improved

**Comment by ta2-1** [comment]
gtm

**Pull request merged**